plugins/janus_streaming.c

```diff
@@ -590,6 +625,11 @@ int janus_streaming_init(janus_callbacks *callback, const char *config_path) {
 		return -1;
 	}
 
+	struct ifaddrs * ifas = NULL;
+	if(getifaddrs(&ifas) || ifas == NULL) {
+		JANUS_LOG(LOG_ERR, "Unable to acquire list of network devices/interfaces; some configurations may not work as expected...\n");
```

Comment:
Well it means the OS is not returning any network interfaces, i.e. claiming there *are* not network interfaces or we're not able to see them for some other reason. This is one of those things that one might file under "should not ever happen".